### PR TITLE
Add demo

### PR DESCRIPTION
**Pull request by Anna-sr**

### CODE REVIEW

File: demo/app.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    model.eval()
+    return model
+
+class App(object):
+    def __init__(self):
+        self.conf_threshold = 0.5
+        self.model = None
+        self.classes = []
+        self.input_height = 512
+        self.input_width = 512
```
Comment (Barsegh-A) [inline demo/app.py:34]: we train model on 256x256, lets do the inference on it as well

File: demo/app.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    def process_image(self, bytes_array):
+        transform = T.Compose([
+            T.Resize((self.input_height, self.input_width)),
+            T.ToTensor()
+        ])
+        labels = inference(bytes_array, self.model, transform, self.conf_threshold)
```
Comment (Barsegh-A) [inline demo/app.py:41]: shouldn't inference be updated to get the image and not the path?

Comment (Barsegh-A) [reply demo/app.py:41]: No :D

File: demo/app.py (hunk 1 above)
Comment (Anna-sr) [reply demo/app.py:34]: ok, will fix it

**Review by Barsegh-A — APPROVED** [review]

**Review by vzohranyan — APPROVED** [review]

**Pull request merged**